modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,74 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
+
+[NOTE]
+====
+For this procedure to run successfully, your `referencePolicy` object must be set to the `Source` type in your `imagestreams.yaml` file:
+[source,yaml]
+----
+referencePolicy:
+    type: Source 
+----
+====
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* You can change the default `importMode` option by using the following example cli artifacts for the image stream: 
```

Comment:
This point might not be super clear. 

The command bellow is an example of how one can patch a given image stream tag (cli-artifacts:latest in the example) to use the PreserveOriginal import mode. Right @Prashanth684 @flavianmissi ?

Reply:
hmm...correct..we probably need to make it clear that this procedure is used to patch an already existing imagestream tag to import manifestlists.